Assets/BossRoom/Scripts/Client/UI/Lobby/LobbyPanelUI.cs

```diff
@@ -0,0 +1,40 @@
+using BossRoom.Scripts.Shared.Infrastructure;
+using BossRoom.Scripts.Shared.Net.UnityServices.Lobbies;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace BossRoom.Scripts.Client.UI
+{
+    /// <summary>
+    /// An individual Lobby in the list of avaialble lobbies
+    /// </summary>
+    public class LobbyPanelUI : ObserverBehaviour<LocalLobby>
+    {
+        public UnityEvent<LocalLobby> OnClicked;
```

Comment:
Again, could be a Unity vs. .NET naming difference, but the convention for this would be `OnClick` instead of the past tense `OnClicked`